source/tutorials/explore-change-streams.txt

```diff
@@ -0,0 +1,230 @@
+.. _kafka-tutorial-explore-change-streams:
+
+==============================
+Explore MongoDB Change Streams
+==============================
+
+Follow this tutorial to learn how to create a change stream on a MongoDB
+collection and observe the change events it creates.
+
+If you already understand how to create change streams and the change
+event data they create, proceed to the
```

Comment:
Suggestion: Double use of create was a bit hard for me to parse, consider replacing second "create". 

```suggestion
If you already understand how to create change streams and the change
event data they contain, proceed to the
```

Reply:
Good catch. I think I will use the verb "produce" as usually you talk about change streams producing or emitting events, not containing them.